src/packages/templating/modals/templating-item-picker/templating-item-picker-modal.element.ts

```diff
@@ -67,14 +70,16 @@ export class UmbTemplatingItemPickerModalElement extends UmbModalBaseElement<
 				pickableFilter: (item) => item.unique !== null,
 			},
 		});
-		await dictionaryItemPickerModal?.onSubmit();
-
-		const dictionaryItem = dictionaryItemPickerModal?.getValue().selection[0];
-
-		if (dictionaryItem) {
-			this.value = { value: dictionaryItem, type: CodeSnippetType.dictionaryItem };
-			this.modalContext?.submit();
-		}
+		await dictionaryItemPickerModal
```

Comment:
Consider using the `.catch(() => undefined)` pattern on this line only and leave the rest be.